bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -59,6 +60,11 @@
      *
      * @return the response body or <code>NULL</code> when the request went wrong
      */
+
+    private static final String USERNAME_KEY = "user";
+    private static final String PASSWORD_KEY = "pass";
+    private static final int DEFAULT_NUMBER_OF_RETRIES = 3;
```

Comment:
I see that you try to keep the old functionality in place, but I actually wonder, if an automatic retry in this util class makes sense - I never saw this being useful anywhere yet and retry handling might better be left to the caller. What do others think?